### PR TITLE
fix: keep @ts- comments

### LINKED ISSUE #377: Do not remove `// @ts-ignore` comments in zap
As title.

### PR DESCRIPTION
**Pull request by lino-levan**
Closes #377.

### CODE REVIEW

File: src/fast_check/transform.rs [hunk 1]
```diff
@@ -110,8 +110,10 @@ impl CommentsMut {
       comments.retain(|_key, value| {
         value.retain(|c| {
           match c.kind {
-            // only keep js docs and @deno-types comments
-            CommentKind::Line => c.text.starts_with("@deno-types"),
+            // only keep js docs, @ts-*, and @deno-types comments
+            CommentKind::Line => {
+              c.text.starts_with("@deno-types") || c.text.starts_with("@ts-")
```
Comment (dsherret) [inline src/fast_check/transform.rs:115]: Oh, this actually might need to use `.trim_start()` first for both of these I think.

**Review by dsherret — COMMENTED** [review]
Thanks! Would you be able to add a test in:

https://github.com/denoland/deno_graph/tree/main/tests/specs/graph

1. Copy and paste one of the other files like _FastCheck_ to a new file.
2. Modify the source to add an example that uses these comments.
3. Run with `UPDATE=1 cargo test --all-features` to update the spec file output and see if it worked.

File: src/fast_check/transform.rs (hunk 1 above)
Comment (lino-levan) [reply src/fast_check/transform.rs:115]: We should definitely have a test for that.

**Review by dsherret — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**